tests/utilities/mock/Scuba.js

```diff
@@ -31,21 +36,21 @@ class Scuba {
             if (!this.supportsInflight) {
                 bucketName = req.params.bucket?.split('_')[0];
                 return res.status(200).json({
-                    bytesTotal: this._data.bucket.get(bucketName)?.current || 0,
+                    bytesTotal: (this._data.bucket.get(bucketName)?.current || 0n).toString(),
                 });
             }
-            const inflight = Number(req.body?.inflight) || 0;
+            const inflight = BigInt(req.body?.inflight || 0);
             this._updateData({
                 action: req.body?.action,
                 bucket: bucketName,
                 inflight,
             });
-            const immediateInflights = req.body?.action === 'objectRestore' ? 0 : inflight;
+            const immediateInflights = req.body?.action === 'objectRestore' ? 0n : inflight;
             return res.json({
-                bytesTotal: (this._data.bucket.get(bucketName)?.current || 0) +
-                    (this._data.bucket.get(bucketName)?.nonCurrent || 0) +
-                    (this._data.bucket.get(bucketName)?.inflight || 0) +
-                    immediateInflights,
+                bytesTotal: ((this._data.bucket.get(bucketName)?.current || 0n) +
+                    (this._data.bucket.get(bucketName)?.nonCurrent || 0n) +
+                    (this._data.bucket.get(bucketName)?.inflight || 0n) +
+                    immediateInflights).toString(),
```

Comment:
do we need toString() here? Is it not the purpose of `BigInt.prototype.toJSON` above? or should we remove that, if it does not work or is not needed?